src/OpenDiffix.Core/CommonTypes.fs

```diff
@@ -375,6 +381,55 @@ module AnonymizationParams =
       | Some tableSettings -> tableSettings.AidColumns |> List.exists (String.equalsI columnName)
       | None -> false
 
+  /// Fails if any of the anon params does not meet the requirements. Set `strict` to `true` to enforce
+  /// checking if the parameters ensure safe minimum level of anonymization, `false` only for basic checks.
+  let validate anonParams =
+    if anonParams.StrictCheck then
```

Comment:
Maybe have 2 anonymous records with min values instead of 2 branches? The latter can become out of date more easily.

Reply:
hm hm, I can try, but this will be more glue code, and what if we introduce a param which must not be greater than x; then the name like `StrictMinValues` will be out of date. Also if we introduce requirements which aren't simple inequalities, etc. I dunno

Reply:
:+1: I just did the refactored version, but I'll undo it and keep in stash. I think I do prefer the dumb way for now.